### PR TITLE
Clarify nuget.exe list protocol

### PR DESCRIPTION
**Pull request by joelverhagen**
Polish on top of https://github.com/NuGet/docs.microsoft.com-nuget/pull/1292.

### CODE REVIEW

File: docs/faqs/NuGet-FAQ.md [hunk 1]
```diff
@@ -232,15 +232,17 @@ If that fails, try the [tips mentioned in this StackOverflow post](http://stacko
 
 ## What is the API endpoint for nuget.org?
 
-To use  nuget.org as a package repository with NuGet clients, you would need to use the following V3 API endpoint: 
+To use nuget.org as a package repository with NuGet clients, you would need to use the following V3 API endpoint:
```
Comment (loic-sharma) [inline docs/faqs/NuGet-FAQ.md]: Thoughts on `you would need to use` -> `you should use` / `you need to use`?

Comment (joelverhagen) [reply docs/faqs/NuGet-FAQ.md]: `you should use` is best I think.

Comment (anangaur) [reply docs/faqs/NuGet-FAQ.md]: I like “you need” better

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[f6d0bcf](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1299/commits/f6d0bcf8ea878ee9550bb63b4ec2eb8fc8c83cc0)_: 

:clock10: **Preparing**: average preparing time is _40 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[f6d0bcf](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1299/commits/f6d0bcf8ea878ee9550bb63b4ec2eb8fc8c83cc0)_: 

:clock10: **Incremental building**: average incremental building time is _21 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[f6d0bcf](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1299/commits/f6d0bcf8ea878ee9550bb63b4ec2eb8fc8c83cc0)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/faqs/NuGet-FAQ.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-nelist/docs/faqs/NuGet-FAQ.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/faqs/nuget-faq?branch=pr-en-us-1299) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C4%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201902041853237619-1299%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=vlQbLjINy8zxgXwhqzbrn%2FMuVyJxBrJlVKTSLWSb7pU%3D&st=2019-02-04T18%3A49%3A48Z&se=2019-03-07T18%3A54%3A48Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/faqs/NuGet-FAQ.md (hunk 1 above)
Comment (joelverhagen) [reply docs/faqs/NuGet-FAQ.md]: You don't need to use. 3.x and new support the V2 protocol. People know this.

File: docs/faqs/NuGet-FAQ.md [hunk 2]
```diff
@@ -232,15 +232,17 @@ If that fails, try the [tips mentioned in this StackOverflow post](http://stacko
...
 **`https://api.nuget.org/v3/index.json`**
 
-Older clients can still use the v2 protocol to reach nuget.org. However, please note, NuGet clients 3.0 or later will have slower and less-reliable service using the v2 protocol:
+Older clients can still use the V2 protocol to reach nuget.org. However, please note, NuGet clients 3.0 or later will have slower and less-reliable service using the V2 protocol:
 
-`https://www.nuget.org/api/v2/` (DEPRECATED!!!) **Note:** the "www." is important.
+`https://www.nuget.org/api/v2/` (DEPRECATED!!!) **Note:** use "www." for the best reliability.
 
-Additionally, *NuGet.exe List* only works with the v2 protocol.
+Additionally, *nuget.exe list*, *push* and *delete* work using V2 endpoints whether the client is configured
```
Comment (loic-sharma) [inline docs/faqs/NuGet-FAQ.md]: While this is  fine, the intent here was to clarify that the `nuget list` command is broken if a server implements just documented APIs (V3 + V2 publish APIs). I think that this new paragraph no longer conveys that. Thoughts?

Comment (joelverhagen) [reply docs/faqs/NuGet-FAQ.md]: This isn't strictly true. You could say the endpoints used by nuget.exe list are both V2 and V3 endpoints. The fact that they are OData via a resource called "LegacyGallery" is an implementation detail. What real life scenario are we trying to protect users from?

Comment (joelverhagen) [reply docs/faqs/NuGet-FAQ.md]: Additionally, we could decouple the endpoints simply by modifying our V3 service index to point to new locations.

Comment (rrelyea) [reply docs/faqs/NuGet-FAQ.md]: if you wanted to have some docs for server implementers, that kind of detail would be fine there? this FAQ is meant to cover how nuget client users should configure the client.
the fact that push, delete and list all get redirected to v2 URIs under the covers is likely irrelevant. I could live without any mention of this fact in this section of the doc.
THoughts?

Comment (anangaur) [reply docs/faqs/NuGet-FAQ.md]: I agree. I would remove the V2 references from this doc. They may be applicable in the API docs not here so much.

Comment (joelverhagen) [reply docs/faqs/NuGet-FAQ.md]: Sure, i'll remove this sentence entirely and enhance the API docs to talk about the `nuget.exe list` case.

Comment (rrelyea) [reply docs/faqs/NuGet-FAQ.md]: k. sounds good.

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[b43f43d](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1299/commits/b43f43dfb7561fb124a1fe90b9fceea3283c3877)_: 

:clock10: **Preparing**: average preparing time is _40 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[b43f43d](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1299/commits/b43f43dfb7561fb124a1fe90b9fceea3283c3877)_: 

:clock10: **Incremental building**: average incremental building time is _21 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[b43f43d](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1299/commits/b43f43dfb7561fb124a1fe90b9fceea3283c3877)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/api/overview.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-nelist/docs/api/overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/overview?branch=pr-en-us-1299) |
[docs/faqs/NuGet-FAQ.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-nelist/docs/faqs/NuGet-FAQ.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/faqs/nuget-faq?branch=pr-en-us-1299) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C4%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201902042134383497-1299%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=yJnU5Y0j2cSwDgDhf43LMr2PhDjPw95EYWt5ewCjUoA%3D&st=2019-02-04T21%3A30%3A46Z&se=2019-03-07T21%3A35%3A46Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[7e2213d](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1299/commits/7e2213de8013bdc36272bcb27cd24f3e33eb7979)_: 

:clock10: **Preparing**: average preparing time is _40 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[7e2213d](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1299/commits/7e2213de8013bdc36272bcb27cd24f3e33eb7979)_: 

:clock10: **Incremental building**: average incremental building time is _21 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[7e2213d](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1299/commits/7e2213de8013bdc36272bcb27cd24f3e33eb7979)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/api/overview.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-nelist/docs/api/overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/overview?branch=pr-en-us-1299) |
[docs/faqs/NuGet-FAQ.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-nelist/docs/faqs/NuGet-FAQ.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/faqs/nuget-faq?branch=pr-en-us-1299) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C4%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201902042157277962-1299%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=CYQ0q48Qeq%2BmKW4CvFcyzf%2FbAuzBaqpG6LKbdqZ0aYk%3D&st=2019-02-04T21%3A53%3A40Z&se=2019-03-07T21%3A58%3A40Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/api/overview.md [hunk 3]
```diff
@@ -79,6 +79,26 @@ that the implementation does not understand should be ignored.
...
+
+Second, we don't document resources that never shipped in an RTM version of the official client.
+`PackageDisplayMetadataUriTemplate` and `PackageVersionDisplayMetadataUriTemplate` fall into this category.
+
+Thirdly, we don't document resources that are tightly coupled with the V2 protocol, which itself is intentionally
+undocumented. The `LegacyGallery` resource falls into this category. This resource allows a V3 service index to point to
+a corresponding V2 source URL. This resource supports the `nuget.exe list`.
+
+If a resource is not documented here, we *strongly* recommend that you do not take a dependency on them. We may remove
+or change the behavior of these undocumented resources which could break your implementation in unexpected ways.
```
Comment (loic-sharma) [inline docs/api/overview.md]: 💯

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[5bcb463](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1299/commits/5bcb4634e0524ad185debb3ef6692df2b323872a)_: 

:clock10: **Preparing**: average preparing time is _40 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[5bcb463](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1299/commits/5bcb4634e0524ad185debb3ef6692df2b323872a)_: 

:clock10: **Incremental building**: average incremental building time is _21 sec(s)_

**Comment by karann-msft** [comment]
OPS Build status updates of commit _[5bcb463](https://github.com/NuGet/docs.microsoft.com-nuget/pull/1299/commits/5bcb4634e0524ad185debb3ef6692df2b323872a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/api/overview.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-nelist/docs/api/overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/api/overview?branch=pr-en-us-1299) |
[docs/faqs/NuGet-FAQ.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-nelist/docs/faqs/NuGet-FAQ.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/faqs/nuget-faq?branch=pr-en-us-1299) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C4%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C201902042223068200-1299%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=vfsQks%2BonNWvGR7w5hFcuz9iv1iUxij%2Fl6gQ5LiVDF8%3D&st=2019-02-04T22%3A19%3A25Z&se=2019-03-07T22%3A24%3A25Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**